qa/tasks/ses_qa.py

```diff
@@ -146,13 +146,18 @@ def drive_replace_initiate(self, **kwargs):
         In order to "hide" an existing disk from the ceph.c_v in teuthology
         the disk is formatted and mounted.
         """
-        # :TODO: make this random and find correct disk device
-        osd_id = "2"
+        from random import randint
+
         total_osds = self.master_remote.sh(number_of_osds_in_cluster)
+        osd_id = randint(0, int(total_osds) - 1)
         assert int(total_osds) == 4, "Unexpected number of osds {} (expected 4)".format(total_osds)
```

Comment:
It is not related to this particular patch but "4" is kind of hardcoded value and it is used in another function driver_replace_check, can we compute total number of osds from the spec of this test.

Reply:
It's tricky:
```
self.ctx['config']['tasks'][22]['deepsea']['drive_group']['custom']['data_devices']['limit']
``` 
gives you `4`

Reply:
Again, you have magic hardcoded values, like '22', and there is no checking the total disk amount, what if it is not 5?
In fact you need this:
```
drive_group = next(x for x in self.ctx['config']['tasks'] \
                   if 'deepsea' in x and 'drive_group' in x['deepsea'])
disks = drive_group['deepsea']['drive_group']['custom']['data_devices']['limit']
```